src/index.html

```diff
@@ -16,17 +19,44 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="paige__body">
+    <div class="search-wrapper">
+      <form
+        class="search-bar"
+        data-qa="big"
+      >
+        <label
+          for="big-search"
+          hidden
+        >
+          Search
+        </label>
+        <input
+          type="text"
+          id="big-search"
+          placeholder="Try 'Los Angeles'"
+          class="search-bar__input big"
+          data-qa="keypress"
+        />
```

Comment:
The class name 'big' does not represent the meaning of the content, it seems to represent a style. Use class names that reflect the content or function rather than the appearance.